src/managers/auth-manager.js

```diff
@@ -247,6 +322,18 @@ AuthManager.prototype = $.extend(AuthManager.prototype, {
         return $d.promise();
     },
 
+    /**
+     * Returns session information for the current user, including the `userId`, `account`, `project`, and `auth_token` (user access token).
+     *
+     * By default, session information is stored in a cookie in the browser. You can change this with the `store` configuration option. 
+     *
+     * **Example**
+     *
+     *      authMgr.getCurrentUserSessionInfo();
+     *
+     * **Parameters**
+     * @param {Object} `options` (Optional) Overrides for configuration options.
+     */
     getCurrentUserSessionInfo: function (options) {
```

Comment:
We should note that unlike most of the other methods, this one is synchronous (i.e., doesn't need `then` or callbacks)